transports/interface.go

```diff
@@ -70,6 +70,9 @@ type AdminService interface {
 	AdminGetPaymail(ctx context.Context, address string) (*buxmodels.PaymailAddress, ResponseError)
 	AdminGetPaymails(ctx context.Context, conditions map[string]interface{}, metadata *buxmodels.Metadata, queryParams *QueryParams) ([]*buxmodels.PaymailAddress, ResponseError)
 	AdminGetPaymailsCount(ctx context.Context, conditions map[string]interface{}, metadata *buxmodels.Metadata) (int64, ResponseError)
+	// AdminCreatePaymail Create a paymail.
+	//
+	// Paymail address (ie. example@bux.org)
```

Comment:
I'm not sure why the comment is necessary for the `AdminCreatePaymail` when, at the same time, other methods (also public) are without a comment.

Besides that, the multiline comment looks to me like a separator between two groups of methods.

Reply:
Comment should show an example of paymail address. Removed new line

Reply:
I've added this to show the example for users how paymail should look like.
![Screenshot 2024-02-08 at 2 45 43 PM](https://github.com/BuxOrg/go-buxclient/assets/150656925/800116ef-6a92-429a-a08b-2ac55fb4ef1e)

Reply:
Because of the task is about just adding a comment/desc to `AdminCreatePaymail ` - agreed, it's ok 👍 

But I'm seeing some inconsistency, why some methods are commented and other not. But maybe it will be resolved by next tasks in the future.